daemon/api_test.go

```diff
@@ -237,19 +245,6 @@ func (s *apiBaseSuite) TearDownSuite(c *check.C) {
 }
 
 func (s *apiBaseSuite) systemctl(args ...string) (buf []byte, err error) {
-	s.sysctlArgses = append(s.sysctlArgses, args)
-
-	if len(args) > 2 && args[0] == "--root" && args[2] == "is-enabled" {
-		// drop the first 2 args which are "--root some-dir"
-		args = args[2:]
-	}
-
-	switch args[0] {
-	case "show", "start", "stop", "restart", "is-enabled":
-	default:
-		panic(fmt.Sprintf("unexpected systemctl call: %v", args))
-	}
-
 	if len(s.sysctlErrs) > 0 {
```

Comment:
I see still usages of sysctlBufs but not Errs, maybe I'm not saying them. If its not used though we should remove it

Reply:
Thanks, missed that. Removed.